crates/runtime/tests/models/openai.rs

```diff
@@ -153,22 +90,22 @@ mod nsql {
 
             let test_cases = [
                 TestCase {
-                    name: "basic",
```

Comment:
Another way to have the snapshots prefixed with openai is to change the module from `nsql` [here](https://github.com/spiceai/spiceai/pull/3727/files?diff=split&w=0#diff-03045e056979f2a71965a2dd7fd5de17313cd746ca68e51400dda723fdd0461bL49) and have it as openai.

Reply:
good idea yeah, there's definitely many tasks we could